modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CrystalClientCodegen.java

```diff
@@ -890,6 +888,7 @@ public String escapeUnsafeCharacters(String input) {
 
     @Override
     public void postProcessFile(File file, String fileType) {
+        super.postProcessFile(file, fileType);
```

Comment:
a few were missing super calls. The super class only has a logger statement in it currently.